pkg/mover/chart.go

```diff
@@ -183,23 +181,19 @@ func NewChartMover(req *ChartMoveRequest, opts ...Option) (*ChartMover, error) {
 		}
 	}
 
-	var err error
-	cm.rawHints, err = loadHints(req.Source.ImageHintsFile, cm.chart, cm.logger)
-	if err != nil {
+	if err := cm.loadHints(&req.Source); err != nil {
 		return nil, err
 	}
-	if cm.rawHints == nil {
-		return nil, ErrImageHintsMissing
-	}
+
 	imagePatterns, err := internal.ParseImagePatterns(cm.rawHints)
 	if err != nil {
 		return nil, fmt.Errorf("failed to parse image patterns: %w", err)
 	}
 
 	cm.logger.Println("Computing relocation...\n")
-	imageChanges, err := cm.pullOriginalImages(imagePatterns)
+	imageChanges, err := cm.loadImages(imagePatterns)
```

Comment:
what about keeping the "original" key in the name?

Reply:
ok

Reply:
done